docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   app:
-    image: golang:1.16.4
+    image: golang:1.18
```

Comment:
I had to update because statticheck was complaining about a new directive that is only present in new Go versions. Also, we were using an old version for testing the library.